src/pages/[...slug].tsx

```diff
@@ -124,6 +125,7 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (
 
   return {
     props: {
+      ...(await serverSideTranslations(locale!, "common")), // load i18n required namespace for all pages
```

Comment:
Building locally I could confirm that every page is loading all the namespaces by analyzing the generated output and the return of this function.

Not sure exactly why but it seems that we need to pass an array of the namespaces here in order to load just the one we want. If you pass a string, it loads everything 🤷🏼 

```suggestion
      ...(await serverSideTranslations(locale!, ["common"])), // load i18n required namespace for all pages
```

Reply:
Thanks for the fix!